### ISSUE TITLE
self-contained: problem with audio tag and opus files

### ISSUE THREAD

**Issue by geh** [jgm/pandoc#4198]
This happens in pandoc 2.0.5 compiled with pandoc-types 1.17.3.

When running: `pandoc -f html -i a.html -t html5 -o b.html -s --self-contained`

With the contents of a.html being:

~~~
<img src=example.jpg />
<audio controls="1" src=example.mp3 />
<audio controls="1" src=example.opus />
~~~

The output b.html file contains the jpg file correctly inlined but none of the audio tags are present, which is strange since pandoc's source code in the SelfContained module indicates that audio tags are supported.

Now if a.html contains:

~~~
<img src=example.mp3 />
~~~

Then it is converted to <embed src="data:audio/Mpeg;base64..." > in b.html, which is fine for me.

But this is not the case with the following:

~~~
<img src=example.opus />
~~~

In which case pandoc says: "Could not determine mime type for `example.opus' ".

**Comment by geh** [comment]
A little bit of progress : the disappearing of <audio> tags happens without --self-contained too.

**Comment by mb21** [comment]
btw, pandoc only supports a subset of html, so if you’re going from html to html, there are better tools for that...

**Comment by jgm** [comment]
I've added opus to the mime types list.
The other thing you need to do is use `-f html+raw_html`, as noted by @mb21.
With the dev version and this change, it should work.

**Comment by jgm** [comment]
Closed by b5ae5373213634a92cb9cca6ae17e8df995e246c

**Comment by geh** [comment]
Thanks a lot for the indications and the quick fix!
